src/Raven.Server/Documents/Subscriptions/Processor/DatabaseSubscriptionProcessor.cs

```diff
@@ -88,8 +88,8 @@ protected override SubscriptionBatchItem GetBatchItem(T item)
                 return batchItem;
             }
 
-            if (Logger.IsInfoEnabled)
-                Logger.Info(reason, batchItem.Exception);
+            if (Logger.IsDebugEnabled)
+                Logger.Debug($"Subscription '{Connection.Options.SubscriptionName}' skipped a document, the reason is: {reason}", batchItem.Exception);
```

Comment:
Just to make sure. This change will handle both logs:

> {DOC-ID} filtered out by criteria

> document '{DOC-ID}' is in status Conflict ...

right?

Reply:
yes those are the `reason` string